src/replication.c

```diff
@@ -1859,6 +1935,11 @@ void replicationAttachToNewMaster(void) {
     freeReplicationBacklog(); /* Don't allow our chained replicas to PSYNC. */
 }
 
+/* Used by replica, returns true whether the connection used for RDB only */
+int isRdbConnectionSync(connection *conn) {
```

Comment:
not sure this function is needed, but i suppose it can help improve readability a bit.
in which case, let's drop the `sync` part (it's not about the operation mode, it's about what this `conn` is used for)
WDYT?
```suggestion
int isSecondChannelConnection(connection *conn) {
```

Reply:
I agree it isn't about operational mode, renaming to `isRdbConnection`